lib/config/options/index.ts

```diff
@@ -749,8 +749,9 @@ const options: RenovateOptions[] = [
   {
     name: 'baseBranches',
     description:
-      'An array of one or more custom base branches to be processed. If left empty, the default branch will be chosen.',
+      'List of one or more custom base branches defined as exact strings (e.g. `["main"]`) and/or via regex expressions (e.g. `["/^release\\/.*/"]`).',
```

Comment:
```suggestion
      'List of one or more custom base branches defined as exact strings or regex expressions',
```

Reply:
Semantically, _and/or_ would make it more explicit that it's not the one or the other (+ also used this way in the rest of the docs). 

Personally, I would prefer the sentence that includes the examples for it's consistency with [matchBaseBranches](https://docs.renovatebot.com/configuration-options/#matchbasebranches) and because the regex format may not be intuitive (written within double quotes and `/` as the only valid delimiter). 

A compromise (in case the above arguments were not convincing) could be this:
```suggestion
      'List of one or more custom base branches defined as exact strings and/or via regex expressions.',
```

Reply:
use a short variant here and add the longer explanation to the markdown file.